src/main/java/org/nest/units/unitrepresentation/SIData.java

```diff
@@ -61,7 +61,7 @@
   private static String[] SIUnitsDerivedRaw =
       {"Hz","N","Pa","J","W","C","V","F","Ohm","S","Wb","T","H","lm","lx","Bq","Gy","Sv","kat"};
   private static String[] SIPrefixesRaw =
-      {"da","h","k","M","G","T","P","E","Z","Y","d","c","m","mu","n","p","f","a","z","y"};
+      {"k","M","G","T","P","E","Z","Y","m","mu","n","p","f","a","z","y"};
```

Comment:
Ok, I see the problem!
I don't think I've ever seen these magnitudes used, except for space and pressure... I have no definite opinion on this, but maybe @jougs or @heplesser will...